tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -0,0 +1,244 @@
+                          "type": "string"
+                        },
+                        "fileName": {
+                          "type": "string"
+                        },
+                        "path": {
+                          "type": "string"
+                        },
+                        "url_path": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "duplicate_found": {
+                          "type": "string"
+                        },
+                        "size": {
+                          "type": "integer"
+                        }
+                      },
+                      "required": [
+                        "key",
+                        "bucket",
+                        "fileName",
+                        "path",
+                        "url_path",
+                        "type",
+                        "duplicate_found",
+                        "size"
+                      ]
+                    },
+                    {
+                      "type": "object",
+                      "properties": {
+                        "key": {
+                          "type": "string"
+                        },
+                        "bucket": {
+                          "type": "string"
+                        },
+                        "fileName": {
+                          "type": "string"
+                        },
+                        "path": {
+                          "type": "string"
+                        },
+                        "url_path": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "duplicate_found": {
+                          "type": "string"
+                        },
+                        "size": {
+                          "type": "integer"
+                        }
+                      },
+                      "required": [
+                        "key",
+                        "bucket",
+                        "fileName",
+                        "path",
+                        "url_path",
+                        "type",
+                        "duplicate_found",
+                        "size"
+                      ]
+                    }
+                  ]
+                }
+              },
+              "required": [
+                "granuleId",
+                "dataType",
+                "version",
+                "files"
+              ]
+            }
+          ]
+        }
+      },
+      "required": [
+        "granules"
+      ]
+    },
+    "config": {
+      "type": "object",
+      "properties": {
+        "glacier-bucket": {
+          "type": "string"
+        },
+        "protected-bucket": {
```

Comment:
See comments on allowing for better bucket handeling. I feel protected, internal, private, and public bucket config items could be replaced with a single buckets config item to allow for more robust code.